src/main/java/com/android/volley/toolbox/HurlStack.java

```diff
@@ -92,7 +92,7 @@ public HttpResponse executeRequest(Request<?> request, Map<String, String> addit
         URL parsedUrl = new URL(url);
         HttpURLConnection connection = openConnection(parsedUrl, request);
         for (String headerName : map.keySet()) {
-            connection.addRequestProperty(headerName, map.get(headerName));
+            connection.setRequestProperty(headerName, map.get(headerName));
```

Comment:
FYI, technically this one seems like a no-op, because it's a brand new connection and so the connection will presumably not have any properties set. But it's fine to change. Only the one below where the content-type property is set after this is a problem. But it's fine to change for consistency.